proposals/0004-register-types-and-diagnostics.md

```diff
@@ -61,6 +61,10 @@ responsible for validating the semantic meaning behind the application of the
 attribute, while the rest of `handleResourceBindingAttr` is responsible for
 validating the syntax of the attribute.
 
+The `register` annotation may also have a second parameter, `spaceN`, where `N` is 
+some integer. `DiagnoseHLSLRegisterAttribute` will also be responsible for validating 
```

Comment:
Any limitation on the `N`?

Reply:
It looks like N may be from 0-42949672879. 
4294967280-4294967295 are reserved for for tooling / OS, according to:
https://microsoft.github.io/DirectX-Specs/d3d/ResourceBinding.html#additional-root-argument-capacity-for-system-instrumentation